### DISCUSSION TITLE
use BP_LAUNCHPOINT for generated files

### DISCUSSION THREAD

**Discussion by gliheng** [paketo-buildpacks/feedback#199] [Node.js Team]
I tried to use a generated js file as BP_LAUNCHPOINT. After hours of work, I found out it's not working and the readme of node-start buildpack also mention that they check that BP_LAUNCHPOINT pointed file should exist. I think this check is not useful but misleading, considering ts, webpack etc. all use generated js files. It's quite normal launchpoint file does not exist before packing. Maybe you should consider remove it?

**Comment by sophiewigmore** [comment]
@paketo-buildpacks/nodejs-maintainers
